modules/airgap/instance/instance_server.tf

```diff
@@ -24,7 +24,7 @@ resource "aws_instance" "master" {
   vpc_security_group_ids = [var.sg_id]
   key_name               = var.key_name
   tags = {
-    Name                 = "${var.resource_name}-server${count.index + 1}"
+    Name                 = "${var.resource_name}-distros-server${count.index + 1}"
```

Comment:
Suggest adding a new tag like `Team = distros` which will not make the name longer